plugins/embed-optimizer/tests/bootstrap.php

```diff
@@ -7,3 +7,6 @@
 
 // Require the suggested plugin.
 require_once __DIR__ . '/../../optimization-detective/load.php';
```

Comment:
Other places where `__DIR__ . '../../'` is used we can also replace with:

```suggestion
require_once TESTS_REPO_ROOT_DIR . '/plugins/optimization-detective/load.php';
```

Reply:
Note that this manual loading of Optimization Detective is done for Embed Optimizer but not for Image Prioritizer because the latter has Optimization Detective as an explicit dependency, whereas the former it is a recommended dependency. The test bootstrap logic automatically loads any explicit dependencies.

Reply:
Here are the relevant instances (_updated_):

```
plugins/optimization-detective/tests/test-uninstall.php:27:		require __DIR__ . '/../../../plugins/optimization-detective/uninstall.php';
plugins/embed-optimizer/tests/bootstrap.php:9:require_once __DIR__ . '/../../optimization-detective/load.php';
plugins/speculation-rules/tests/test-speculation-rules-uninstall.php:27:		require __DIR__ . '/../../../plugins/speculation-rules/uninstall.php';
```

Reply:
Oh, the lines in `auto-sizes` are obsolete. They should have been removed with #1476

Reply:
> Oh, the lines in `auto-sizes` are obsolete. They should have been removed with #1476

Removed in 6421043c3d9a4bedeadb77311e20e13b9498ccdf